python/amici/petab_objective.py

```diff
@@ -721,20 +719,20 @@ def rdatas_to_measurement_df(
         measurement_df: pd.DataFrame) -> pd.DataFrame:
     """
     Create a measurement dataframe in the PEtab format from the passed
-    `rdatas` and own information.
+    ``rdatas`` and own information.
 
     :param rdatas:
         A sequence of rdatas with the ordering of
-        `petab.get_simulation_conditions`.
+        :func:`petab.get_simulation_conditions`.
```

Comment:
Another example of a PEtab library method that isn't linked

https://amici--1817.org.readthedocs.build/en/1817/generated/amici.petab_objective.html#amici.petab_objective.rdatas_to_measurement_df

Reply:
Yes, I noticed that as well (#1818). Will check separately.